.travis.yml

```diff
@@ -3,13 +3,14 @@ language: php
 sudo: false
 
 php:
-  - 5.6
-  - 7.0
-  - 7.1
   - 7.2
+  - 7.3
+  - 7.4
+  - 8.0snapshot
```

Comment:
Surprised that Travis doesn't have a proper PHP8 thing already!

Reply:
Last time I checked, they don't! But soon I hope 👍

Reply:
We have to wait the [`php-build`](https://github.com/php-build/php-build) 1.0.0 release I think...
References:
* https://travis-ci.community/t/php-8-0-missing/10132
* https://github.com/php-build/php-build/pull/651